README.md

```diff
@@ -27,7 +27,11 @@ You can retrieve binaries for Linux, macOS and Windows on our [GitHub release pa
 ##### Linux
 If you prefer to do it straight from CLI here's a one-liner for you:
 ```bash
-curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0-alpha.0/opencompose-v0.1.0-alpha.0-d0edfd9-linux-64bit.tar.xz | tar -xJf - -C ${HOME}/bin ./opencompose
+curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0-alpha.0/opencompose-v0.1.0-alpha.0-d0edfd9-linux-64bit.tar.xz | tar -xJ ./opencompose
+```
+and if you want it installed system-wide:
+```bash
+sudo mv ./opencompose /usr/local/bin/
```

Comment:
@cdrage not missing `+x`. The file is from tar and it already has all the permissions set up correctly.

> specifying the file output (don't want users to override something by accident)

I can't really see how would they do that. Would you mind giving an example?
(For me it's shorter and nicer the way it is but if there is a way to get it easily wrong I can change it.)